README.md

```diff
@@ -22,10 +22,15 @@ custom:
   client:
     bucketName: unique-s3-bucketname-for-your-website-files
     distributionFolder: client/dist # (Optional) The location of your website. This defaults to client/dist
+    indexDocument: index.html # (Optional) The location of your index document. Defaults to index.html
+    errorDocument: error.html # (Optional) The location of your error document. Defaults to error.html
+    spa: false # (Optional) Set this for single-page applications to redirect all errors to the index document
```

Comment:
@evanseeds Is there a benefit to adding this extra configuration value over just letting the user set both the index and error document to index.html? 

It seems like it's an extra config value? Here's what I mean:

This:
```
indexDocument: index.html
spa: true
```

Is the same expected outcome as this:
```
indexDocument: index.html
errorDocument: index.html
```

Any objection to removing the spa option? I'm open to it with valid use-case. I might just not understand it presently.

Reply:
My use case is using finch to deploy single-page applications that need all traffic redirected to index.html. This allows the webapp in the index.html file to handle all routes. I think it's more intuitive to be able to just set a 'spa' flag to true. Otherwise, a user would be to have to figure out how to implement this feature in S3, and then how to translate those S3 settings into finch.

In most of these cases you would be using the default 'index.html' indexDocument, and so the config comparison would be:
```
spa: true
```

vs
```
indexDocument: index.html
errorDocument: index.html
```

The specific changes being made are more obvious in the second, but I think having the first option is more user-friendly.

'spa' may not be the best name, for sure. 'singlePage' or 'singlePageApplication' would be clearer names. Alternative, a more descriptive name like 'redirectErrorsToIndex' may work too.

Ultimately, there's a part of me that just doesn't like having to write the same thing twice, but maybe that's just an obsession 😅